pkg/apis/build/v1alpha2/image_types.go

```diff
@@ -51,6 +51,7 @@ type ImageSpec struct {
 	ProjectDescriptorPath    string                            `json:"projectDescriptorPath,omitempty"`
 	Build                    *corev1alpha1.ImageBuild          `json:"build,omitempty"`
 	Notary                   *corev1alpha1.NotaryConfig        `json:"notary,omitempty"`
+	DefaultProcess           string
```

Comment:
Curious to know when this object gets marshalled and why this new field is missing json format annotations (or is it just overlooked)?

Reply:
Overlooked, good catch!